e2e/testsuite/testsuite.go

```diff
@@ -533,3 +553,32 @@ func getValidatorsAndFullNodes(chainIdx int) (int, int) {
 	tc := LoadConfig()
 	return tc.GetChainNumValidators(chainIdx), tc.GetChainNumFullNodes(chainIdx)
 }
+
+// GetMsgTransfer returns a MsgTransfer that is constructed based on the channel version
+func GetMsgTransfer(portID, channelID, version string, tokens sdk.Coins, sender, receiver string, timeoutHeight clienttypes.Height, timeoutTimestamp uint64, memo string) *transfertypes.MsgTransfer {
```

Comment:
Added this helper function to construct the `MsgTransfer` depending on the transfer app channel version.